test/testSetup.ts

```diff
@@ -56,8 +57,28 @@ export class TestSetup {
     await this.checkForUncaughtErrors();
     try {
       await utilities.deleteScratchOrg(this.scratchOrgAliasName, this.reuseScratchOrg);
+      await this.deleteScratchOrgInfo();
     } catch (error) {
-      utilities.log(`Deleting scratch org failed with Error: ${(error as Error).message}`);
+      utilities.log(
+        `Deleting scratch org (or info) failed with Error: ${(error as Error).message}`
+      );
+    }
+  }
+
+  private async deleteScratchOrgInfo(): Promise<void> {
+    if (this.scratchOrgId) {
+      const sfDataDeleteRecord = await utilities.runCliCommand(
+        'data:delete:record',
+        '--sobject',
+        'ScratchOrgInfo',
+        '--where',
+        `Id=${this.scratchOrgId}`
+      );
+      if (sfDataDeleteRecord.exitCode > 0) {
+        const message = `data delete record failed with exit code ${sfDataDeleteRecord.exitCode}\n stderr ${sfDataDeleteRecord.stderr}`;
+        utilities.log(message);
+        throw new Error(message);
```

Comment:
Why should this cause a test failure?

Reply:
the function is called inside a try block so it wouldn't cause the failure! I can just remove it too 🙂

Reply:
I like removing the try/catch

Reply:
I'm lost. Removing the try/catch in tearDown that we just added in a recent PR? 🧠